### PR TITLE
update README, use next tag for npm, yarn installation

### PR DESCRIPTION
**Pull request by sibiraj-s**
use next tag to always install the latest version of bootstrap published to the registry

### CODE REVIEW

**Comment by Zain3** [comment]
Mute the thread

**Comment by mdo** [comment]
I feel like I'd rather be specific in our versions. Thoughts @XhmikosR?

**Comment by XhmikosR** [comment]
Yup, agreed.

File: README.md [hunk 1]
```diff
@@ -40,8 +40,8 @@ Several quick start options are available:
 
 - [Download the latest release.](https://github.com/twbs/bootstrap/archive/v4.0.0-beta.2.zip)
```
Comment (bkeeton12) [inline README.md]: Download

**Review by bkeeton12 — APPROVED** [review]

**Comment by Sibiraj-S** [comment]
I felt the same @mdo when I used to see the versions get updated manually. 

But this will be those installations, however, points to latest(beta) one. then why not make use of npm's @next tag? I am not sure how the composer works though.

**Comment by Sibiraj-S** [comment]
something like in the last [commit](https://github.com/twbs/bootstrap/pull/24644/commits/e71df55b378ab4739b75ba04f552a13882a9acbc).

but if it is GTG with the current format. it is A-Ok then

**Comment by XhmikosR** [comment]
Thanks, but we'd rather be explicit currently. Remember, v4-dev will be merged to master eventually.

**Comment by Sibiraj-S** [comment]
Cool then

**Pull request closed without merging**